package.json

```diff
@@ -7,6 +7,7 @@
     "build": "vite build",
     "preview": "vite preview",
     "test": "playwright test",
+    "test-html": "playwright test --reporter=html",
```

Comment:
I also added this script so we don't have to remember the command to show an html report of the diffs instead of scrolling through the terminal output

Reply:
oh thank god